<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>376</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>737</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>R.E.P.O SAVE MANAGER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="default_path_label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:12px;</string>
        </property>
        <property name="text">
         <string>C:/Users/User/AppData/LocalLow/semiwork/REPO/saves</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="saveName_">
             <property name="styleSheet">
              <string notr="true">font-size:16px;</string>
             </property>
             <property name="text">
              <string>R.E.P.O</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="saveLevel_">
             <property name="styleSheet">
              <string notr="true">font-size:16px;</string>
             </property>
             <property name="text">
              <string>Level : 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="savePlayTime_">
             <property name="styleSheet">
              <string notr="true">font-size:16px;</string>
             </property>
             <property name="text">
              <string>Game Time : 0h</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="saveDate_">
             <property name="styleSheet">
              <string notr="true">font-size:16px;</string>
             </property>
             <property name="text">
              <string>2025-06-10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QListWidget" name="saveListB">
           <property name="minimumSize">
            <size>
             <width>231</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QListWidget" name="namesListB">
           <property name="minimumSize">
            <size>
             <width>171</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="backupButton">
             <property name="minimumSize">
              <size>
               <width>151</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>51</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size:14px;</string>
             </property>
             <property name="text">
              <string>Create Backup</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="restoreButton">
             <property name="minimumSize">
              <size>
               <width>151</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>51</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size:14px;</string>
             </property>
             <property name="text">
              <string>Restore Selected Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="restoreallButton">
             <property name="minimumSize">
              <size>
               <width>151</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>51</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size:14px;</string>
             </property>
             <property name="text">
              <string>Restore All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="minimumSize">
              <size>
               <width>151</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>51</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size:14px;</string>
             </property>
             <property name="text">
              <string>Delete All</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="game_path_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:16px;</string>
        </property>
        <property name="text">
         <string>Change game's default save path</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
